### PR TITLE
Delete stale comment on waiting for ok-to-test from member.

### PR DESCRIPTION
**Pull request by xiangpengzhao**
Fixes: https://github.com/kubernetes/test-infra/issues/5221

/cc @cjwagner @kargakis @cblecker

### CODE REVIEW

File: prow/plugins/trigger/pr.go [hunk 1]
```diff
@@ -57,6 +58,10 @@ func handlePR(c client, trustedOrg string, pr github.PullRequestEvent) error {
 			return fmt.Errorf("could not validate PR: %s", err)
 		} else if trusted {
 			c.Logger.Info("Starting all jobs for updated PR.")
+			err = clearStaleComments(c.GitHubClient, trustedOrg, pr, nil)
```
Comment (cjwagner) [inline prow/plugins/trigger/pr.go]: `trustedPullRequest` and `clearStaleComments` will both list the issue comments right now. Please list the issue comments once and pass them to these two functions. Then this change won't increase API token usage except for the actual comment deletion if it is needed.

Comment (xiangpengzhao) [reply prow/plugins/trigger/pr.go]: Done. Your concern on reducing API token usage always makes sense :+1:

**Comment by xiangpengzhao** [comment]
@cjwagner updated. PTAL.

File: prow/plugins/trigger/pr.go [hunk 2]
```diff
@@ -52,17 +53,29 @@ func handlePR(c client, trustedOrg string, pr github.PullRequestEvent) error {
...
-		trusted, err := trustedPullRequest(c.GitHubClient, pr.PullRequest, trustedOrg)
+		comments, err := c.GitHubClient.ListIssueComments(pr.PullRequest.Base.Repo.Owner.Login, pr.PullRequest.Base.Repo.Name, pr.PullRequest.Number)
+		if err != nil {
+			return err
+		}
+		trusted, err := trustedPullRequest(c.GitHubClient, pr.PullRequest, trustedOrg, comments)
 		if err != nil {
 			return fmt.Errorf("could not validate PR: %s", err)
 		} else if trusted {
 			c.Logger.Info("Starting all jobs for updated PR.")
```
Comment (cjwagner) [inline prow/plugins/trigger/pr.go]: nit: Move this log line to above `return buildAll(c, pr.PullRequest)`

**Comment by xiangpengzhao** [comment]
@cjwagner inline comments addressed. PTAL. Thanks!

**Pull request merged**